tests/basics/class_dict.py

```diff
@@ -0,0 +1,10 @@
+
+class Foo:
+    self.a = 1
+    self.b = "bar"
```

Comment:
`self` shouldn't be needed here

Reply:
whoops, fixed along with splitting the commits.